components/progressTracker/theme.css

```diff
@@ -64,9 +64,39 @@
     }
   }
 
-  &:not(.is-active):not(.is-completed) {
-    .progress-step__label {
-      opacity: 0.5;
+  .status-bullet {
+    background-color: transparent;
+    transition: background-color .1s var(--animation-curve-default) calc(var(--animation-duration) - .1s);
+    align-self: center;
+    border-radius: 50%;
+    display: block;
+    height: 12px;
+    margin: 6px 0;
+    width: 12px;
+    z-index: 2;
+    position: relative;
+    &:before {
+      background-color: transparent;
+      transition: background-color 6s var(--animation-curve-default) calc(var(--animation-duration) - .1s);
```

Comment:
6 seconds? 🤔

Reply:
forgot to switch this one out.

 :ok: by @Kemosabert on 19/06/2018 09:53:33